cfme/automate/import_export.py

```diff
@@ -39,10 +36,8 @@ def is_displayed(self):
 
 
 class AutomateImportExportView(AutomateImportExportBaseView):
-    @View.nested
-    class flash(V2VFlashMessages):
-        ROOT = 'div.import-flash-message'
 
+    flash = View.nested(ImportExportFlashMessages)
```

Comment:
I don't think you need to define this in here as the `AutomateImportExpoertView` will inherit the `flash` from the `AutomateImportExportBaseView`